pkg/utils/utils.go

```diff
@@ -15,6 +15,8 @@ var (
 	NetDirectory = "/sys/class/net"
 	// SysBusPci is sysfs pci device directory
 	SysBusPci = "/sys/bus/pci/devices"
+	// UserspaceDrivers is a list of driver names that don't have netlink representation for their devices
+	UserspaceDrivers = []string{"vfio-pci", "uio_pci_generic"}
```

Comment:
I *think* that's the right name for UIO driver but I am not sure. Perhaps someone with uio experience can clarify.

Reply:
It is right one. We also need to include "igb_uio" there.